woodwork/tests/utils/test_accessor_utils.py

```diff
@@ -37,6 +37,12 @@ def test_init_series_valid_conversion_specified_ltype(sample_series):
     assert series.ww.semantic_tags == set()
 
 
+def test_init_series_with_invalid_type(sample_df):
+    error_message = f'Input must be of series type. The current input is of type {type(sample_df)}'
+    with pytest.raises(TypeError, match=error_message):
+        init_series(sample_df)
```

Comment:
I like the idea of testing a DataFrame here! 

Two other cases that would be nice to have here as well are some simpler object like an `int` or a `str` as well as `None` which are other possible accidental inputs to `init_series`.

Reply:
Are there any fixtures for those two data types that I can use?

Reply:
Nope, we generally use fixtures when we need to build an object up for a test, which lets us avoid repetitive code, but something like `1` or `None` is just an object that doesn't need any initialization, so you can just use it directly

Reply:
Alright I've added checks for those three datatypes